.github/workflows/build-zip-file.yml

```diff
@@ -2,15 +2,40 @@ name: "Build zip file"
 
 # This action will run when it is triggered manually.
 on:
+  workflow_call:
+    inputs:
+      skip-build-zip:
+        type: boolean
+        required: false
+        default: false
+        description: "Skip running build zip"
+      skip-smoke-tests:
+        type: boolean
+        required: false
+        default: false
+        description: "Skip running smoke tests"
+      repo-branch:
```

Comment:
@jbordonado I understand it's always best to keep inputs to minimum. However, in this case, the caller workflow & build zip are 2 different workflows / jobs and hence to checkout a specific branch, that needs to be passed as an input.

I'm not sure whether it's possible to checkout a specific branch on one job/workflow & use it on another one without passing it as an input. For instance, the release branch is created from `develop` by the [release - code freeze workflow](https://github.com/Automattic/woocommerce-payments/actions/workflows/release-code-freeze.yml) and then calls the build zip workflow. If we don't pass the branch name to build zip, it would run on `develop`. 

Let me know your thoughts.

Reply:
You're definitely right. I had something else in mind, but it looks like it's not possible actually.